src/main/java/seedu/cc/logic/commands/entrylevel/FindCommand.java

```diff
@@ -59,7 +68,7 @@ public FindCommand(ExpenseDescriptionContainsKeywordsPredicate expensePredicate,
     @Override
     public CommandResult execute(Model model, ActiveAccount activeAccount) {
         requireAllNonNull(model, activeAccount);
-        assert (expensePredicate != null || revenuePredicate != null);
+        assert (activeAccount != null);
```

Comment:
this needs to be  assert (expensePredicate != null || revenuePredicate != null); the one you give already covered in requiredAllNonNull. At this point, it can't be both predicates are null.